config.go

```diff
@@ -1,6 +1,6 @@
 package specs
 
-// Spec is the base configuration for the container.  It specifies platform
+// Spec is the base configuration for the container. It specifies platform
```

Comment:
in contrast, a sentence with more than one sentence _would_ have ending punctuation.